core/templates/dev/head/pages/base.html

```diff
@@ -165,6 +165,38 @@ <h2>
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <!-- About-page -->
+    <script src="/templates/dev/head/pages/about/about-page.module.js"></script>
```

Comment:
Ditto.

Reply:
I am not sure, (comment link https://github.com/oppia/oppia/pull/6524#discussion_r278119285). I will try to remove them and push the changes if those pass tests then ill keep them removed.

Reply:
done